tests/ballerina-compiler-api-test/src/test/java/io/ballerina/semantic/api/test/symbols/EnumDeclSymbolTest.java

```diff
@@ -146,7 +146,7 @@ public void testEnumMemberDeclarations(int line, int col, String expName, String
 
         // check type
         assertEquals(symbol.typeKind(), TypeDescKind.SINGLETON);
-        assertEquals(symbol.typeDescriptor().typeKind(), typeDescKind);
+        assertEquals(symbol.typeDescriptor().typeKind(), TypeDescKind.SINGLETON);
```

Comment:
@pubudu91, FYI.

Reply:
Ack. Shall we properly update this test case by getting rid of the `typeDescKind` param and updating the data provider as well?